services/deck.js

```diff
@@ -157,6 +162,38 @@ const self = module.exports = {
 
 };
 
+async function createImages(content, sourceURL, targetURL, authtoken){
+    if(content.includes('src="https://fileservice.')){//only execute if images are in the slide
+        let $ = cheerio.load(content);
+        let urls = $(content).find('img').map(async (i, image) => {//process each image in the slide
+            let src = URL.parse($(image).attr('src'));
+            if(src.host.includes('fileservice.')) {//only process images from the fileservice
```

Comment:
We need to explicitly set the target fileservice URL as it may be anything really, instead of relying on the target deck service URL to have any particular pattern. The command becomes bigger but it's an admin tool that can be scripted, so we don't really care about that :)

Reply:
Is this OK?